src/Skia/Avalonia.Skia/FormattedTextImpl.cs

```diff
@@ -207,6 +207,10 @@ public override string ToString()
                 try
                 {
                     ApplyWrapperTo(ref currentPaint, foreground, ref currd, paint);
+
+                    paint.SubpixelText = true;
```

Comment:
This is already set there, I think: https://github.com/JurjenBiewenga/Avalonia/blob/64d01ae9054253b4e5b9e9fd2c71a10611192e4b/src/Skia/Avalonia.Skia/FormattedTextImpl.cs#L40

Reply:
LcdRenderText has to be set to false, I removed my code and moved it there.